app/src/components/map/MapContainer.tsx

```diff
@@ -105,6 +203,32 @@ const MapContainer: React.FC<IMapContainerProps> = (props) => {
     // eslint-disable-next-line react-hooks/exhaustive-deps
   }, [preDefinedGeometry]);
 
+  useEffect(() => {
+    if (!geometryState?.geometry.length || !nonEditableGeometries?.length) {
+      setInferredLayersInfo &&
+        setInferredLayersInfo({
+          parks: [],
+          nrm: [],
+          env: [],
+          wmu: []
+        });
+    }
+
+    const isAllPointGeometries = geometryState?.geometry.every((geo: Feature) => geo.geometry.type === 'Point');
+
+    if (isAllPointGeometries) {
+      throttledGetFeatureDetails(layersToInfer);
+    }
+  }, [geometryState?.geometry, nonEditableGeometries]);
+
+  useEffect(() => {
+    if (!bounds || !bounds.length) {
+      return;
+    }
+
+    throttledGetFeatureDetails(layersToInfer);
+  }, [bounds]);
```

Comment:
Any reason why this wouldn't work? I tried it locally and it seems to work, though I don't know all of the edge cases to test, so maybe you can verify it.

```suggestion
  useEffect(() => {
    if (!geometryState?.geometry.length || !nonEditableGeometries?.length) {
      setInferredLayersInfo &&
        setInferredLayersInfo({
          parks: [],
          nrm: [],
          env: [],
          wmu: []
        });
    }

      throttledGetFeatureDetails(layersToInfer);
  }, [geometryState?.geometry, nonEditableGeometries]);
```

Reply:
Roughly the idea being that we update the feature details on geometry change, rather than bounds + all points (which seemed like a bit of an odd set of triggers for this effect).

Reply:
great idea!